www/src/pages/en/t3-collection.md

```diff
@@ -11,7 +11,7 @@ Made a project using the T3 stack and want to share it? Add it to the list!
 | Description                                                       | Repo                                                            | Link                                                                  |
 | ----------------------------------------------------------------- | --------------------------------------------------------------- | --------------------------------------------------------------------- |
 | Create T3 Turbo - T3 Stack using Turborepo                        | [create-t3-turbo](https://github.com/t3-oss/create-t3-turbo)    | [create-t3-turbo.vercel.app](https://create-t3-turbo.vercel.app/)     |
-| Zapdos - a QnA app for streamers                                  | [t3-oss/zapdos](https://github.com/t3-oss/zapdos)               | [z.t3.gg](https://z.t3.gg)                                            |
+| Zapdos - a QnA app for streamers                                  | [t3-oss/zapdos](https://github.com/t3-oss/zapdos)               | [ask.ping.gg](https://ask.ping.gg)                                    |
```

Comment:
repo is moved to pingdotgg so can change that while we're at it

Reply:
https://github.com/pingdotgg/zapdos

Reply:
Done.